hack/utils/daemon.ml

```diff
@@ -173,10 +173,8 @@ let spawn
           fn)  in
   let out_fd =
     Unix.openfile out_path [Unix.O_WRONLY; Unix.O_CREAT; Unix.O_TRUNC] 0o666 in
-  let pid =
-    Unix.create_process
-      Sys.executable_name [|Sys.executable_name|]
-      null_fd out_fd out_fd in
+  let exe = Sys_utils.executable_path () in
```

Comment:
this is so we use an absolute path to the same binary, instead of relying on PATH. sometimes `Sys.executable_name` is not correct, like if you do `PATH='~/flow/bin/' flow` (where the path is in single quotes so that the shell doesn't expand it)